avaframe/out3Plot/plotUtils.py

```diff
@@ -135,6 +136,15 @@
 colorsS = ['#FFCEF4', '#FFA7A8', '#C19A1B', '#578B21', '#007054', '#004960', '#201158']
 cmapS = cmapCameri.batlow.reversed()
 
+# colormap used if no resType provided
+cmapNNcmap = cmapCameri.imola.reversed()
+
+# colormap for probabilities
+levProb = [0.0, 0.25, 0.5, 0.75, 0.9999999999, 10]
```

Comment:
see issue #488  related to the 0.99999

Reply:
I add fixing levels for probability colormap in the issue